setup.py

```diff
@@ -36,6 +36,7 @@ def read(fname):
     },
     long_description=read("README.md"),
     long_description_content_type="text/markdown",
+    ext_modules=[Extension("bitsandbytes", sources=[], language="c")],
```

Comment:
Is there actually a Python extension module by the name of `bitsandbytes`?  If not, I don't think this is correct?

Reply:
> Is there actually a Python extension module by the name of `bitsandbytes`? If not, I don't think this is correct?

there is no actual extension module named `bitsandbytes`, but how do we distinguish a linux bin and a win bin?
with this fix, we can obtain both `bitsandbytes-0.42.0-cp310-cp310-win_amd64.whl` and `bitsandbytes-0.42.0-cp310-cp310-linux_x86_64..whl`. without this fix, we can get only one `bitsandbytes-*-py3-none-any.whl` wheel.

Reply:
Right, so the intent is to signal to the build machinery that there is native code in here and the wheel should be tagged as such 😄 

That wasn't clear from the (lack of 😉) comment or the commit message "add ext_modules" :)

Let's at least make the intent clear here so whoever will port this to a non-setup.py build knows what's up 😄 

```suggestion
    # HACK: pretend we have a native extension module so the wheel is tagged 
    #       correctly with a platform tag (e.g. `-linux_x86_64.whl`).
    ext_modules=[Extension("bitsandbytes", sources=[], language="c")],
```